lib/kvdb/kvdb_home.c

```diff
@@ -234,5 +234,12 @@ kvdb_home_get_config(const char *home, cJSON **conf)
         return merr(EBADMSG);
     }
 
+    /* If the file doesn't exist, it isn't an error. */
+    if (access(conf_file_path, F_OK) == -1) {
+        if (errno == ENOENT)
+            return 0;
+        return merr(errno);
+    }
+
```

Comment:
Why not just try the open and handle it there?  Less code...

Reply:
The code that this calls is also shared by the hse.conf code path